Atmosphere/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <heading class="container">
+        <section class="container-section">
+        <h4>Ipsum dolor sit amet</h4>
+        <h1>MAGNA SED CONSEQUAT</h1>
+        <a href="#" class="btn">get started</a>
+    </section>
+    </heading>
+    <main>
+        <article class="container-card">
+            <section class="card">
+             <h4>magna</h4>
+             <h2>Dolor sit amet vivamus</h2>
+             <p>Phasellus in ante vitae nibh porttitor tempus vitae ut ante. Vestibulum blandit ex a elementum viverra. Sed quat diam, aliquet tempus felis nisl at lacus.</p>
+             <a href="#" class="btn-one">more</a>
+            </section>
+            <section class="card">
+           <h4>LOREM</h4>
+            <h2>Ipsum dolor sit amet</h2>
+            <p>Phasellus in ante vitae nibh porttitor tempus vitae ut ante. Vestibulum blandit ex a elementum viverra. Sed quat diam, aliquet tempus felis nisl at lacus.</p>
+            <a href="#" class="btn-one">more</a>   
+            </section>
+        </article>
+        <article class="description">
+            <section>
+                <div>
+                    <i class="fa-solid fa-chart-column post-icons"></i>
+                </div>
+               <div>
+                <h4>PLACERAT</h4>
+                <P>Lorem ipsum dolor sit amet.nullam et consequate</P>
+               </div>
+               <div>
+                <i class="fa-brands fa-telegram post-icons"></i>
+                 <div>
+                  <h4>LIBERO</h4>
+                  <p>Lorem ipsum dolor sit amet, consectetur</p>
+                 </div>
+                 <div>
+                    <i class="fa-solid fa-chart-area post-icons"></i>
+                    
+                 </div>
+                 <div>
+                    <h4>SOLICITU</h4>
+                    <p>Lorem ipsum dolor sit amet, consectetur</p>
+                 </div>
+                 <div>
+                    <i class="fa-solid fa-image post-icons"></i> 
+                 </div>
+                 <div>
+                    <h4>TEMPOR</h4>
+                    <p>Lorem ipsum dolor sit amet, consectetur</p>
+                 </div>
+            </section>
+        </article>
+
+        <article class="img-article">
+            <section class="class-img">
+              <h4>CONSEQUAT</h4>
```

Comment:
maring is too much check for and font sizing and correct the font color spacing etc